packages/sdk/python/human-protocol-sdk/human_protocol_sdk/escrow.py

```diff
@@ -0,0 +1,516 @@
+
+        Args:
+            address (str): The Ethereum address associated with the key-value pair
+            key (str): The key of the key-value pair to get
+
+        Returns:
+            None
+
+        Raises:
+            EscrowClientError: If an error occurs while checking the parameters
+        """
+
+        if not Web3.isAddress(escrow_address):
+            raise EscrowClientError("Invalid address")
+        if not hash:
+            raise EscrowClientError("Invalid hash")
+        if not URL(url):
+            raise EscrowClientError("Invalid URL")
+        if not self.w3.eth.default_account:
+            raise EscrowClientError("You must add an account to Web3 instance")
+
+        self._handle_transaction(
+            "Store Results",
+            self._get_escrow_contract(escrow_address).functions.storeResults(
+                self.w3.eth.default_account, url, hash
+            ),
+        )
+
+    def complete(self, escrow_address: str):
+        """Sets the status of an escrow to completed.
+
+        Args:
+            escrow_address (str): Address of the escrow to complete
+
+        Returns:
+            None
+
+        Raises:
+            EscrowClientError: If an error occurs while checking the parameters
+        """
+
+        if not Web3.isAddress(escrow_address):
+            raise EscrowClientError("Invalid address")
+
+        self._handle_transaction(
+            "Complete", self._get_escrow_contract(escrow_address).functions.complete()
+        )
+
+    def bulk_payout(
+        self,
+        escrow_address: str,
+        recipients: List[str],
+        amounts: List[Decimal],
+        final_results_url: str,
+        final_results_hash: str,
+        txId: Decimal,
+    ):
+        """Pays out the amounts specified to the workers and sets the URL of the final results file.
+
+        Args:
+            escrow_address (str): Address of the escrow
+            recipients (List[str]): Array of recipient addresses
+            amounts (List[Decimal]): Array of amounts the recipients will receive
+            final_results_url (str): Final results file url
+            final_results_hash (str): Final results file hash
+
+        Returns:
+            None
+
+        Raises:
+            EscrowClientError: If an error occurs while checking the parameters
+        """
+
+        if not Web3.isAddress(escrow_address) or not all(
+            Web3.isAddress(recipient) for recipient in recipients
+        ):
+            raise EscrowClientError("Invalid address")
+        if len(recipients) != len(amounts):
+            raise EscrowClientError("Arrays must have same length")
+        if len(recipients) == 0:
+            raise EscrowClientError("Arrays must have any value")
+        if 0 in amounts:
+            raise EscrowClientError("Amounts cannot be empty")
+        if sum(amounts) > self.get_balance(escrow_address):
+            raise EscrowClientError("Escrow does not have enough balance")
```

Comment:
Let's change it to `f"Escrow does not have enough balance. Current balance: {balance}. Amounts: {sum(amounts)}"`